### ISSUE TITLE
broadcast: display last x number of notify messages on the broadcast UI

### ISSUE THREAD

**Issue by saxon-milton** [ausocean/cloud#184]
Would this be useful ?

**Comment by scruzin** [comment]
Yes, it would be useful, but I view this as a search feature, i.e., similar to searching for log messages. There could of course be a quick link from the broadcast page to the search page.

**Comment by scruzin** [comment]
Regardless of UI choice, it depends on Issue #187
